example/src/Examples/Reanimated/index.tsx

```diff
@@ -4,7 +4,7 @@ import { ScrollView } from "react-native-gesture-handler";
 
 import { AnimateTextOnPath } from "./AnimateTextOnPath";
 import { AnimationWithTouchHandler } from "./AnimationWithTouchHandler";
-import { BokehExample } from "./BokehExample";
+//import { BokehExample } from "./BokehExample";
```

Comment:
Maybe leave out changes in the examples to separate PRs since they're not related to this change, otherwise document why.